feat(serverless): add serverless job retries MTA-6904 (#6602)
scaleway/docs-content: pages/serverless-jobs/concepts.mdx

Refer to the [dedicated FAQ](/serverless-jobs/faq/#how-can-i-configure-access-to-a-private-network-or-virtual-private-cloud-vpc) for more information on Serverless Jobs compatibility with Private Networks and [Virtual Private Cloud (VPC)](/vpc/).

## Retries

> Retries refer to the automatic re-execution of a job run that has failed. Serverless Jobs can be configured to automatically retry a job run after up to five failed attempts. [Learn how to manage the retry attempts of a Serverless Job.](/serverless-jobs/how-to/manage-job-retries/)

Reviewer: ```suggestion
Retries refer to the automatic retry of a failed job run. You can configure Serverless Jobs to automatically retry a failed job run, up to five times. [Learn how to manage the retry attempts of a Serverless Job.](/serverless-jobs/how-to/manage-job-retries/)
```

> Retries refer to the automatic re-execution of a job run that has failed. Serverless Jobs can be configured to automatically retry a job run after up to five failed attempts. [Learn how to manage the retry attempts of a Serverless Job.](/serverless-jobs/how-to/manage-job-retries/)

Author: ```suggestion
Retries refer to the automatic re-execution of a failed job run. You can configure Serverless Jobs to automatically retry a job run after up to five failed attempts. [Learn how to manage the retry attempts of a Serverless Job.](/serverless-jobs/how-to/manage-job-retries/)
```
